### ISSUE TITLE
Types not exported correctly

### ISSUE THREAD

**Issue by racso-dev** [samhirtarif/react-audio-recorder#88]
**Describe the bug**
There seems to be problems with types.

When I import the component and hook from the package, like so 
```
import { AudioRecorder, useAudioRecorder } from 'react-audio-voice-recorder';
```
I have this issue with the import:
```
Could not find a declaration file for module 'react-audio-voice-recorder'. '/home/oscar/Desktop/workspaces/front/node_modules/.pnpm/react-audio-voice-recorder@2.1.2_react-dom@18.2.0_react@18.2.0/node_modules/react-audio-voice-recorder/dist/react-audio-voice-recorder.es.js' implicitly has an 'any' type.
  There are types at '/home/oscar/Desktop/workspaces/front/node_modules/react-audio-voice-recorder/dist/index.d.ts', but this result could not be resolved when respecting package.json "exports".The 'react-audio-voice-recorder' library may need to update its package.json or typings.ts(7016)
```

**Package info (please complete the following information):**
 - Version 2.1.2

**Comment by denkochev** [comment]
Same problem. Waiting for answers.

**Comment by samhirtarif** [comment]
It would be great if either of you could create a minimal reproduction of this problem. Would really help with debugging.

**Comment by samhirtarif** [comment]
I have merged PR #89.
